helm_deploy/values-preprod.yaml

```diff
@@ -6,6 +6,7 @@ generic-service:
 
   ingress:
     host: restricted-patients-api-preprod.hmpps.service.justice.gov.uk
+    contextColour: green
```

Comment:
This is already in values.yaml, but I need to change something to make it deploy